compat/js/siteorigin-panels-layout-block.js

```diff
@@ -278,6 +278,7 @@ registerBlockType('siteorigin-panels/layout-block', {
     var onLayoutBlockContentChange = function onLayoutBlockContentChange(newPanelsData) {
       if (!_.isEmpty(newPanelsData.widgets)) {
         // Send panelsData to server for sanitization.
+        wp.data.dispatch('core/editor').lockPostSaving();
```

Comment:
Do you perhaps want to format this line and the line below with the same spacing as the changes to `siteorigin-panels-layout-block.jsx` ?

Reply:
This file is automatically generated by the build process, and it's removing the spacing inside of the parentheses. I can change its formatting, but it'll be replaced upon running the build process.

Reply:
Thanks for letting me know, yeah, that's no problem then.